ImageNet/train/dawn_4_machine_30min.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "v1.0",
+    "author": "Andrew Shaw, Yaroslav Bulatov, Jeremy Howard",
+    "authorEmail": "andrew.t.shaw@gmail.com",
+    "framework": "ncluster / pytorch",
+    "codeURL": "https://github.com/diux-dev/imagenet18/blob/master/training/train_imagenet_nv.py",
+    "model": "Resnet 50",
+    "hardware": "32 * V100 (4 machines - AWS p3.16xlarge)",
+    "costPerHour": 24.48,
```

Comment:
I think this is the wrong value for `costPerHour`. You used 4 p3.16xlarge machines, so shouldn't the cost be 97.92 (4x24.48)?